src/widgets/_viewswitcher.scss

```
@@ -0,0 +1,103 @@
+viewswitcher {
+    & button {
+        margin: 0;
+        padding: 0;
+        border-radius: 0;
+        border-top: 0;
+        border-bottom: 0;
+        box-shadow: none;
+        font-size: rem(16px);
+
+        &:not(:checked):not(:hover) {
+            background: transparent;
+        }
+
+        &:not(:only-child):not(:last-child) {
+            border-right-width: 0;
+        }
+
+        &:not(only-child):first-child:not(:checked):not(:hover),
+        &:not(:checked):not(:hover) + &:not(:checked):not(:hover) {
+            border-left-color: transparent;
+        }
+
+        &:not(only-child):last-child:not(:checked):not(:hover) {
+            border-right-color: transparent;
+        }
+
+        &:not(:checked):hover:not(:backdrop) {
+            background-image: #{'image(lighter(@color_primary))'};
+        }
+
+        &:not(only-child):first-child:not(:checked):hover,
+        &:not(:checked):hover + &:not(:checked):not(:hover),
+        &:not(:checked):not(:hover) + &:not(:checked):hover {
+            border-left-color: #{'shade(@color_primary, 1.15)'};
+        }
+
+        &:not(only-child):last-child:not(:checked):hover {
+            border-right-color: #{'shade(@color_primary, 1.15)'};
+        }
+
+        &:not(:checked):hover:backdrop {
+            background-image: #{'image(@color_primary)'};
+        }
+
+        stack > box {
+
+            & label.active {
+                font-weight: bold;
+            }
+
+            &.narrow {
+                font-size: rem(12px);
```

This seems like it should probably instead be
```suggestion
                @extend .small-label
```